airflow/jobs/scheduler_job.py

```diff
@@ -1759,3 +1759,49 @@ def _emit_pool_metrics(self, session: Session = None) -> None:
     @provide_session
     def heartbeat_callback(self, session: Session = None) -> None:
         Stats.incr('scheduler_heartbeat', 1, 1)
+
+    @provide_session
+    def reset_state_for_orphaned_tasks(self, session: Session = None):
+        """
+        Reset any TaskInstance still in QUEUED state that were enqueued by a
```

Comment:
```suggestion
        Reset any TaskInstance still in SCHEDULED and QUEUED state that were enqueued by a
```
At least if I correctly understand this query:
```
session.query(TI).filter(TI.state.in_([State.SCHEDULED, State.QUEUED]))
```